.zuul.yaml.bak

```diff
@@ -0,0 +1,46 @@
+- project:
```

Comment:
why we need a bak file?

Reply:
This PR is heavily in progress. I'll change a lot of stuff before it is ready for review